jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -104,7 +106,7 @@ def __init__(self, *args, **kwargs):
             self.spatial_subset = SubsetSelect(self,
                                                'spatial_subset_items',
                                                'spatial_subset_selected',
-                                               default_text='Entire Cube',
+                                               default_text=SPATIAL_DEFAULT_TEXT,
```

Comment:
I don't think we have time for this right now, so I'm just including it here so that it's written down. Ultimately, I could see the actual default being set as fallback logic within `SubsetSelect` such that:
```
if allowed_type == 'spatial':
    self.default_text = 'Entire Cube'
else:
    self.default text='Entire Spectrum'
```

and then later down, it can be referenced by calling `if text_field = SubsetSelect.default_text` or something